app_vue/index.html

```diff
@@ -5,7 +5,7 @@
     <link rel="icon" href="/favicon.ico">
```

Comment:
I think this path needs to be changed to /public/favicon.ico

Reply:
actually either way it should be fine! In vue, anything in the public folder gets straight up copied over to the /dist, so we can refer to them as "/" directly.

Additionally this code is untouched! its the default path setup of Vue for the previous favicon